src/pages/QuestionSetEditor.js

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useRef } from "react";
+import { questionSetEditorConfig } from "../components/data";
```

Comment:
@suvarnakale make this file ts

Reply:
this file is replaced. with Editor.tsx page